blockchain/chain/src/store/chain_store.rs

```diff
@@ -441,6 +526,19 @@ where
         .collect()
 }
 
+pub fn get_parent_reciept<DB>(
```

Comment:
This kinda goes for others as well, but can you add docs on the public methods/types?